lake-primitives/src/types/impl_actions.rs

```diff
@@ -104,7 +104,7 @@ impl Action {
         }
     }
 
-    // Tries to convert a IndexerTransactionWithOutcome to a Vec<Action>
+    // Tries to convert a [IndexerTransactionWithOutcome] to a [Vec<Action>]
```

Comment:
```suggestion
    /// Tries to convert a [IndexerTransactionWithOutcome] to a [Vec<Action>](Action)
```
This will point to the outer most type, i.e. Vec